addons/loyalty/models/loyalty_program.py

```diff
@@ -50,7 +50,8 @@ class LoyaltyProgram(models.Model):
         ('next_order_coupons', 'Next Order Coupons')],
         default='promotion', required=True,
     )
-    date_to = fields.Date(string='Validity')
+    date_from = fields.Date(string='Start Date')
+    date_to = fields.Date(string='End date')
```

Comment:
```suggestion
    date_from = fields.Date(string="Start Date")
    date_to = fields.Date(string="End date")
```
By convention, strings that can be read by the users are between `"`, others by `'`